libs/shared/tsconfig.json

```diff
@@ -1,12 +1,3 @@
 {
-    "compilerOptions": {
-        "target": "ES2020",
-        "jsx": "react",
-        "module": "commonjs",
-        "resolveJsonModule": true,
-        "esModuleInterop": true,
-        "forceConsistentCasingInFileNames": true,
-        "strict": true,
-        "skipLibCheck": true
-    }
+    "extends": "../../tsconfig.base.json",
```

Comment:
Should `"include": ["./src/**/*"]` also be added here?

Reply:
should